File: linear_algebra/Python/UNK_BT8.py
n = int(input("Nhap so nguyen duong n "))
if n <= 0:
    print("n phai la so nguyen duong")
else:
    print('Ma tran 0')
    for i in range(n):
        for j in range(n):
            print(0, end=' ')
        print()
    print('Ma tran don vi')
    for i in range(n):
        for j in range(n):
            if i == j:
                print(1, end=' ')
            else:
                print(0, end=' ')
        print()

File: linear_algebra/C++/UNK_BT8.cpp
#include <iostream>
using namespace std;

int main()
{
    int n;
    cout << "Nhap so nguyen duong n ";
    cin >> n;

    if (n <= 0) {
        cout << "n phai la so nguyen duong";
        return 0;
    }
    int matran0[100][100];
    int matranDV[100][100];

    for (int i = 0; i < n; i++) {
        matranDV[i][i] = 1;
    }

    cout << "Ma tran 0" << endl;
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            cout << matran0[i][j] << " ";
        }
        cout << endl;
    }
    cout << "Ma tran don vi" << endl;
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            cout << matranDV[i][j] << " ";
        }
        cout << endl;
    }
    return 0;
}